google/gapic/longrunning/operations_client.py

```diff
@@ -0,0 +1,279 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
Could you update this manually until we can regenerate the package in `api-client-staging`? (Or, preferably, update the package in `api-client-staging` and copy it back into here.)